doc/source/changes/version_0_26.rst.inc

```diff
@@ -112,6 +112,8 @@ Miscellaneous improvements
     >>> group_even_year
     year[2000, 2002, 2004, 2006, 2008, 2010, 2012, 2014, 2016]
 
+* argument `axes` of `split_axes` has become optional: defaults to the last axis (closes :issue:`365`).
```

Comment:
Sorry. I forgot to update the issue... But I wonder if splitting all axes containing sep wouldn't be a better default.